base/exports.jl

```diff
@@ -993,6 +993,7 @@ export
     @timev,
     @elapsed,
     @allocated,
+    @loadtime,
```

Comment:
What about `@time_imports`?

This only measures timing for serialized code, right? Anything marked `__precompile__(false)` will not be timed.

Reply:
Yeah that's good! thanks